### ISSUE TITLE
Project templates use older SDK version

### ISSUE THREAD

**Issue by FireGiantHelp** [firegiant/HeatWaveSupport#41]
### HeatWave Version

1.0.1

### Visual Studio Version

17.6.1

### Repro Steps

1. Create a new project

### Actual Result

When adding a new project with any WiX (HeatWave for VS2022) template, the project sdk is set at 4.0.0 in the project file.

```xml
<Project Sdk="WixToolset.Sdk/4.0.0">
</Project>
```

### Expected Result

I expected this to be the latest SDK version (4.0.1)

```xml
<Project Sdk="WixToolset.Sdk/4.0.1">
</Project>
```

Original reported at https://github.com/wixtoolset/issues/issues/7543

**Comment by FireGiantHelp** [comment]
This issue is on our backlog to fix.

**Comment by FireGiantHelp** [comment]
We expect this issue is fixed in [HeatWave v1.0.2 now available in Visual Studio Marketplace](https://www.firegiant.com/blog/2023/10/13/heatwave-v1.0.2-and-heatwave-build-tools-released/). Visual Studio should provide notification of the update or [you can update manually](https://www.firegiant.com/docs/heatwave/).
